lib/Extractor/Solver.cpp

```diff
@@ -61,6 +62,9 @@ static cl::opt<bool> InferInsts("souper-infer-inst",
 static cl::opt<bool> EnableExhaustiveSynthesis("souper-exhaustive-synthesis",
     cl::desc("Use exaustive search for instruction synthesis (default=false)"),
     cl::init(false));
+static cl::opt<bool> EnableAliveTest("souper-solver-alive",
+    cl::desc("Use alive for solver tests (default=false)"),
```

Comment:
don't say "for solver tests" since there's nothing specific about testing here

let's have one alive command line option, don't have one here and one in synthesis

Reply:
I mean, the "souper-use-alive" flag should be used here, please do not add another flag for this